bundles/io.openliberty.tools.eclipse.product/about.properties

```
@@ -0,0 +1,6 @@
+blurb=Liberty Tools for Eclipse IDE\n\
+\n\
+Version: {featureVersion}\n\
+\n\
+Copyright (c) 2023 IBM Corporation and others.\n\
```

```suggestion
Copyright (c) 2022, 2023 IBM Corporation and others.\n\
```

Though this part is new, this is describing the feature as a whole, so I guess the <initial,latest> format is better?